src/_path_wrapper.cpp

```diff
@@ -454,16 +454,18 @@ static PyObject *Py_affine_transform(PyObject *self, PyObject *args, PyObject *k
     if (PyArray_NDIM(vertices_arr) == 2) {
         numpy::array_view<double, 2> vertices(vertices_arr);
         npy_intp dims[] = { (npy_intp)vertices.size(), 2 };
+        Py_DECREF(vertices_arr);
```

Comment:
perhaps move it even before declaring/initing `dims[]`?  interleaving it in the middle of creating `results` is a bit weird...
(ditto below)

Reply:
I just misread `vertices` as the thing we were using originally.